docs/en/03_How_tos/akismet.md

```diff
@@ -34,12 +34,13 @@ to store this value.
 You can also add the following YAML code into your mysite/_config/config.yml
 
 
-	:::yml
-	---
-	Name: myspamprotection
-	---
-	AkismetSpamProtector:
-	  api_key: XXXXXXXXXXXXX
+```yml
+---
+Name: myspamprotection
+---
+AkismetSpamProtector:
+  api_key: XXXXXXXXXXXXX
+```
```

Comment:
```yaml
---
Name: myspamprotection
---
SilverStripe\Akismet\AkismetSpamProtector:
  api_key: 5555dddd55d5d
```

Reply:
Shouldn't put your real credentials in the examples, Robbie. 🕺

Reply:
True, crap